messages.js

```diff
@@ -47,6 +47,10 @@ Messages.send = function (target, context) {
 	let buf;
 	if ((target.charAt(0) === '/' && target.charAt(1) !== '/') || (target.charAt(0) === '!' && /[a-z0-9]/.test(target.charAt(1)))) {
 		// PM command
+		if (targetUser === user && target.charAt(0) === '!') {
+			// Don't allow users to PM broadcast themselves commands
+			target = target.replace('!', '/');
```

Comment:
@Slayer95 - I just think of the idea of doing this, but above this line, doing something like:
``user.send('|pm|' + user.getIdentity() + '|' + targetUser.getIdentity() + '|' + target);``

That way, messages would appear as they are being broadcasted, but in actuality, the server made their actual command turn into using ``/`` instead of ``!``, but it would look as if they __were__ being broadcasted. 

Thoughts?